### PR TITLE
Add Radiozenders.FM support

### PR DESCRIPTION
**Pull request by calvinturbo**

### CODE REVIEW

File: src/connectors/radiozenders.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+Connector.playerSelector = '#jp_container_1';
+
+Connector.getArtistTrack = function() {
+	let artistTrack = $('#player-station-info').text();
+	return Util.splitArtistTrack(artistTrack, [' - '], true);
+};
+
+Connector.isPlaying = () => $('.player-equalizer').is(':visible');
```
Comment (inverse) [inline src/connectors/radiozenders.js]:
```suggestion
Connector.playButtonSelector = '.player-equalizer';
```

**Comment by calvinturbo** [comment]
Sorry for the delay but I added the test case and fixed scrobbling for a specific radio station (Q-Music).

File: src/connectors/radiozenders.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+Connector.playerSelector = '#jp_container_1';
+
+Connector.getArtistTrack = function() {
+	let artistTrack = $('#player-station-info').text();
+	return Util.splitArtistTrack(artistTrack, [' - '], reverse);
+};
+
+Connector.playButtonSelector = '.player-equalizer';
```
Comment (inverse) [inline src/connectors/radiozenders.js]: I think this needs some more work. This is used to determine if the player is playing or not. Looking at the site I think we should overload the `Connector.isPlaying` function like this:

```
Connector.isPlaying = () => {
	return $('#jp_container_1').hasClass('jp-state-playing');
};
```

Comment (calvinturbo) [reply src/connectors/radiozenders.js]: Changed it, thanks

**Comment by calvinturbo** [comment]
Changed the isPlaying check, time to merge I think

**Pull request merged**